src/views/Scenario/Scenario.js

```diff
@@ -186,6 +293,7 @@ const Scenario = (props) => {
 };
 
 Scenario.propTypes = {
+  dispatchSetScenarioValidationStatus: PropTypes.func.isRequired,
```

Comment:
It would not be better not to use _dispatch_ prefix in order to standardize with other dispatcher function props ?
Maybe use _setScenarioValidationStatus_ insteed of dispatchSetScenarioValidationStatus ?